srml/council/src/voting.rs

```diff
@@ -129,32 +158,36 @@ decl_event!(
 	pub enum Event<T> where <T as system::Trait>::Hash {
 		/// A voting tally has happened for a referendum cancellation vote.
 		/// Last three are yes, no, abstain counts.
-		TallyCancelation(Hash, u32, u32, u32),
+		TallyCancellation(Hash, u32, u32, u32),
```

Comment:
could we remove comments like "Last three are yes, no, abstain counts." and just define more types to improve developability?

for example, could we add some types like:
```
pub type YaysCount = u32;
pub type NaysCount = u32;
pub type AbstainsCount = u32;
```

then replaces the event with `TallyCancellation(Hash, YaysCount, NaysCount, AbstainsCount),`

then remove the comments.

Note: This would go in a separate PR since this PR is just related to documentation.